docs/BuildONNX.md

```diff
@@ -9,29 +9,39 @@ You will need to install python 3.x if its not default in your environment, and
 You will also need `pybind11` which may need to be installed (mac: `brew install pybind11` or linux: `apt -y install python3-pybind11` for example) and you may need to indicate where to find the software (Mac, POWER, possibly other platforms: `export pybind11_DIR=<your path to pybind>`). Then install the `third_party/onnx` software (Mac: `pip install -e third_party/onnx`) typed in the top directory.
 
  ## Upgrading ONNX in ONNX-MLIR
- 
+``` 
 Here are the steps taken to upgrade the ONNX version:
 
 1.	Create your own branch
 
-2.	cd into `third_party/onnx` and checkout the commit for the latest version of onnx (You can find the latest commit here: https://github.com/onnx/onnx/releases)
+2.	"cd" into `third_party/onnx` and checkout the commit for the latest version of onnx (You can find the latest commit here: https://github.com/onnx/onnx/releases)
 
-3.	pip uninstall onnx (remove older version)
+3.	"pip uninstall onnx" (remove older version)
 
-4.	In `onnx-mlir/` directory, pip install —user third_party/onnx (install onnx from the commit and not online version)
+4.	In `onnx-mlir/` directory, "pip install -e third_party/onnx" (install onnx from the commit and not online version)
```

Comment:
Is option "-e" really needed?

Reply:
Yes! Based on the comment @justinchuby made here: https://github.com/onnx/onnx/issues/5716

`Please make sure the setuptools in the environment is up to date. You should also use pip install -e . for editable mode installs, and use python -m build for building wheels. @liqunfu we should call this out in the release notes`

Reply:
If you don’t intend to build in development mode then you may drop the -e. e is for editable.

Reply:
Thanks for clarification. We are not in development mode for onnx package.